documentation/plugin-dev-guide.html

```diff
@@ -266,137 +314,215 @@ <h2>Modifying the Admin Console</h2>
-the Openfire admin console via a plugin. The general theme is
-that plugins should integrate seamlessly:
-
-<ul>
-        <li>Integrate into existing tabs and sidebar sections whenever possible
-            instead of creating your own. Only create new tabs for very
-            significant new functionality.
-        <li>Don't use the word "plugin" in names of tabs, sidebars and items.
-            For example, instead of having an item called "Gateway Plugin", it
-            could be called "Gateway Settings".
-        <li>Try to match the UI of the existing admin console in your custom
-            plugin pages.
-        <li>There is no need to create an admin console entry to show plugin
-            meta-data. Instead, let Openfire inform the user about which
-            plugins are installed and provide plugin management.
-</ul>
-
-<h3>Writing Pages for the Admin Console</h3>
-
-<p>
-
-Openfire uses the <a href="http://www.opensymphony.com/sitemesh/" target="_blank">Sitemesh</a>
-framework to decorate pages in the admin console. A globally-defined decorator is applied to
-each page in order to render the final output, as in the following diagram:</p>
-<br>
-<div align="center"><img src="images/sitemesh.png" width="484" height="372" alt="Sitemesh"></div>
-<br><br>
-<p>
-Creating pages that work with Sitemesh is easy. Simply create valid HTML pages and then
-use meta tags to send instructions to Sitemesh. When rendering the output, Sitemesh will
-use the instructions you provide to render the decorator along with any content in the
-body of your HTML page. The following meta tags can be used:
-<ul>
-    <li><b>pageID</b> -- the ID of the page, which must match an entry in the admin console
-    XML described above. Either a pageID or subPageID <b>must</b> be specified.</li>
-
-    <li><b>subPageID</b> -- the ID of the sub-page, which must match an entry in the
-    admin console XML described above. Sub-pages are used for administrative actions
-    related to a parent page ID. For example, editing or deleting a particular group.
-    Either a pageID or subPageID <b>must</b> be specified.</li>
-    <li><b>extraParams</b> (Optional) -- extra parameters that should be passed in to the page.
-    For example, on a page to delete a group it might be the ID of the group. Parameters
-    must be URL encoded.</li>
-    <li><b>decorator</b> (Optional) -- overrides the Sitemesh decorator to use for the page.
-    A decorator named <tt>none</tt> is available that will simply render the page
-    without a decorator.</li>
-
-</ul>
-
-The following HTML snippet demonstrates a valid page:
-
-<fieldset>
-    <legend>Sample HTML</legend>
-<pre>
-   &lt;html&gt;
+    </section>
+
+    <section id="writing-pages">
+
+        <h2>Writing Pages for the Admin Console</h2>
+
+        <p>
+            Openfire uses the <a href="https://www.opensymphony.com/sitemesh/" target="_blank">Sitemesh</a>
```

Comment:
Is the OpenSymphony website gone?

Reply:
Yeah, I noticed that too. I don't immediately have a credible replacement.